Program.cs

```diff
@@ -1,7 +1,23 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System;
+
+namespace Training {
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter a string: ");
+         string iso = Console.ReadLine ();
+         bool isogram = Isogram (iso);
+         Console.WriteLine ($"{iso} is {(Isogram (iso) ? "" : "not ")}an isogram.");
+      }
+
+      static bool Isogram (string iso) {
+         iso = iso.ToLower ();
+         HashSet<char> chars = new HashSet<char> ();
+         foreach (char c in iso) {

```

Comment:
You could avoid the curly brace here. Can you think of how to do?

Reply:
Removed curly braces